google/ggrc-core
# GGRC-8243 Migrate html to markdown for Scope objects

PR should be merge in **ggrcq_ggrc_scope_sync** branch

# Issue description
Migrate html to markdown for Scope objects


# Steps to test the changes
Run migration on database dump and check that attributes migrate to markdown without tags

# Solution description
Prepare migration for processing necessary fields from html to markdown

# Sanity checklist


# Migration checklist


# PR Review checklist
